website/addons/dryad/settings/defaults.py

```diff
@@ -0,0 +1,32 @@
+CLIENT_ID = None
+CLIENT_SECRET = None
+
+SCOPE = ['repo']
+
+SET_PRIVACY = False
+
+#DRYAD OAI-PMH metadata harvesting settings
+#Note where requested, format as (date, prefix, dataset)
+DRYAD_OAI_IDENTIFY = "http://www.datadryad.org/oai/request?verb=Identify"
```

Comment:
I saw you're requiring another python module to use as a Dryad API client, and am wondering if these constants are needed here?